### PR TITLE
Fork(2), exit(3), wait4(2): add simple test.

### PR DESCRIPTION
**Pull request by LeeLin2602**
Fork(2), exit(3), wait4(2): add simple test.

### CODE REVIEW

File: tests/sys/syscalls/Makefile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
```
Comment (lwhsu) [inline tests/sys/syscalls/Makefile:1]: superfluous white lines

File: tests/sys/syscalls/fork/Makefile [hunk 2]
```diff
@@ -0,0 +1,12 @@
+
```
Comment (lwhsu) [inline tests/sys/syscalls/fork/Makefile:1]: superfluous white lines

File: tests/sys/syscalls/fork/fork_test.c [hunk 3]
```diff
@@ -0,0 +1,82 @@
+/*-
+ * SPDX-License-Identifier: BSD-2-Clause
+ *
+ * Copyright (c) 2023 Lin Lee <leelin2602@gmail.com>
```
Comment (lwhsu) [inline tests/sys/syscalls/fork/fork_test.c:4]: It's 2024 now.

File: tests/sys/syscalls/fork/fork_test.c [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    ATF_REQUIRE(simple_test(64));
+}
+
+ATF_TP_ADD_TCS(tp) {
+    ATF_TP_ADD_TC(tp, test_simple_test_89);
+    ATF_TP_ADD_TC(tp, test_simple_test_64);
+
+    return atf_no_error();
+}
+
```
Comment (lwhsu) [inline tests/sys/syscalls/fork/fork_test.c:82]: superfluous white lines

File: tests/sys/syscalls/fork/fork_test.c [hunk 5]
```diff
@@ -0,0 +1,82 @@
+/*-
+ * SPDX-License-Identifier: BSD-2-Clause
+ *
+ * Copyright (c) 2023 Lin Lee <leelin2602@gmail.com>
+ *
+ * This software was developed by SRI International and the University of
+ * Cambridge Computer Laboratory under DARPA/AFRL contract FA8750-10-C-0237
+ * ("CTSRD"), as part of the DARPA CRASH research programme.
```
Comment (lwhsu) [inline tests/sys/syscalls/fork/fork_test.c:8]: Did you copy the file from the work of those projects? If so, you should add the copyright lines of them. If no, this part should be removed.

**Review by lwhsu — CHANGES_REQUESTED** [review]
I think the mtree needs updating as well.

File: tests/sys/syscalls/fork/fork_test.c (hunk 5 above)
Comment (igalic) [inline tests/sys/syscalls/fork/fork_test.c:8]: is this just a copy paste again?

**Comment by LeeLin2602** [comment]
Please look into https://github.com/freebsd/freebsd-src/pull/1097.

**Pull request closed without merging**